po/r-novice-gapminder.ja.po

```diff
@@ -828,30 +828,30 @@ msgstr ""
 "---\n"
 "# Please do not edit this file directly; it is auto generated.\n"
 "# Instead, please edit 01-rstudio-intro.md in _episodes_rmd/\n"
-"title: \"Introduction to R and RStudio\"\n"
-"teaching: 45\n"
-"exercises: 10\n"
-"questions:\n"
-"- \"How to find your way around RStudio?\"\n"
-"- \"How to interact with R?\"\n"
-"- \"How to manage your environment?\"\n"
-"- \"How to install packages?\"\n"
-"objectives:\n"
-"- \"Describe the purpose and use of each pane in the RStudio IDE\"\n"
-"- \"Locate buttons and options in the RStudio IDE\"\n"
-"- \"Define a variable\"\n"
-"- \"Assign data to a variable\"\n"
-"- \"Manage a workspace in an interactive R session\"\n"
-"- \"Use mathematical and comparison operators\"\n"
-"- \"Call functions\"\n"
-"- \"Manage packages\"\n"
-"keypoints:\n"
-"- \"Use RStudio to write and run R programs.\"\n"
-"- \"R has the usual arithmetic operators and mathematical functions.\"\n"
-"- \"Use `<-` to assign values to variables.\"\n"
-"- \"Use `ls()` to list the variables in a program.\"\n"
-"- \"Use `rm()` to delete objects in a program.\"\n"
-"- \"Use `install.packages()` to install packages (libraries).\"\n"
+"レッスンタイトル: \"R と RStudio\"\n"
```

Comment:
For consistency.
```suggestion
"タイトル: \"R と RStudio\"\n"
```

Reply:
Actually, remove this as resolved here: https://github.com/masamiy/i18n/pull/1/commits/9cd9432b193f403444f1422563d4269258e2eeb9